Source/dead.cpp

```diff
@@ -76,7 +76,7 @@ void AddDead(Point tilePosition, int8_t dv, Direction ddir)
 	dDead[tilePosition.x][tilePosition.y] = (dv & 0x1F) + (ddir << 5);
 }
 
-void SetDead()
+void SyncUniqDead()

```

Comment:
Also, what does "Sync" mean here? I think we should rename this to something that makes more sense/is more specific.

Reply:
dunno, I literally copy/pasted the name

Reply:
i think its related to loading them from saves